bin/build_flang-legacy.sh

```diff
@@ -45,6 +45,12 @@ else
   _cxx_flag=""
 fi
 
+# Legacy Flang dosen't support building of compiler-rt so it
+# utilizes the clang runtime libraries build/install using build_project.sh.
+# The LLVM_VERSION_MAJOR of legacy flang driver has to match with the clang
+# binaries generated from build_project.sh.
+LLVM_VERSION_MAJOR=`$(AOMP)/bin/clang --version | awk 'NR==1 && match($0, /[0-9]+.[0-9]+.[0-9]+/) {s = substr($0, RSTART, RLENGTH); split(s, a, "."); print a[1]}'
```

Comment:
This is missing an ending back tick \`. Even with that added I see an error:
`Make Error at CMakeLists.txt:48 (project):`
`VERSION ".0.0" format invalid.`

If you just want the major version you can use: `clang --version | grep -oP '(?<=clang version )[0-9]+'`